.github/workflows/ci.yml

```diff
@@ -26,4 +26,4 @@ jobs:
       run: pip install -r requirements.txt
       
     - name: Run tests
-      run: pytest --exhaustive
+      run: python -m pytest --exhaustive
```

Comment:
Out of interest, what's the difference in behaviour here?

Reply:
`-m` always add the current folder to PYTHONPATH, which I needed to import modules from tools/